Lib/asyncio/base_events.py

```diff
@@ -40,6 +40,7 @@
 from . import sslproto
 from . import tasks
 from .log import logger
+from .constants import DEBUG_STACK_DEPTH
```

Comment:
In asyncio we always import modules, so please change to `from . import constants`.

Reply:
like in the line above, `from .log import logger`? :-)

(yes I fixed it)

Reply:
That logger import is probably the only offender, will fix it in a separate PR :)